exchange/bidder.go

```diff
@@ -115,9 +118,27 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, request *openrtb.Bi
 			bids, moreErrs := bidder.Bidder.MakeBids(request, httpInfo.response)
 			errs = append(errs, moreErrs...)
 			for _, bid := range bids {
+				var targets map[string]string
+				targets = nil
+				if bidderTarg.targetFlag {
+					cpm := bid.Bid.Price
+					width := bid.Bid.W
+					height := bid.Bid.H
+					deal := bid.Bid.DealID
+					cacheKey := ""
+					var err error
+					targets, err = bidder.makePrebidTargets(cpm, width, height, cacheKey, deal, bidderTarg)
```

Comment:
Would recommend just passing the full `bid` (pointer, to avoid a copy) here. It's much simpler, and less code.